<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="Sanju" failures="1" tests="2" name="Test(failed)" time="0.052" errors="0" timestamp="2024-11-19T08:30:59 IST">
  <testcase classname="TestNG.ex9_Sample2" name="TC5" time="0.013">
    <failure type="java.lang.AssertionError" message="Failed both results are diff expected [Hello] but found [Hi]">
      <![CDATA[java.lang.AssertionError: Failed both results are diff expected [Hello] but found [Hi]
at TestNG.ex9_Sample2.TC5(ex9_Sample2.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- TC5 -->
  <testcase classname="TestNG.ex9_Sample1" name="TC2" time="0.002"/>
  <testcase classname="TestNG.ex9_Sample1" name="TC3" time="0.0">
    <ignored/>
  </testcase> <!-- TC3 -->
  <testcase classname="TestNG.ex9_Sample2" name="TC6" time="0.0">
    <ignored/>
  </testcase> <!-- TC6 -->
  <testcase classname="TestNG.ex9_Sample2" name="TC4" time="0.0">
    <ignored/>
  </testcase> <!-- TC4 -->
  <testcase classname="TestNG.ex9_Sample1" name="TC1" time="0.0">
    <ignored/>
  </testcase> <!-- TC1 -->
</testsuite> <!-- Test(failed) -->
